domain-eval/README.md

```diff
@@ -0,0 +1,99 @@
+# Domain Specific Evaluation with Argilla, Distilabel, and LightEval
+
+This project demonstrates the creation, annotation, and evaluation of a model on a domain specific task using Argilla, distilabel, and Lighteval. The process involves generating exam questions from multiple documents, annotating them, creating a dataset, and evaluating language models on the task.
```

Comment:
Add the links to each github repo for the lib

Reply:
You also need to homogeneise lighteval citations (we usually do `ligtheval`, but really it's up to you as long at it's consistent)